gnat2goto/driver/tree_walk.adb

```diff
@@ -3634,7 +3539,8 @@ package body Tree_Walk is
    function Do_Operator_General (N : Node_Id) return Irep is
    begin
       if Nkind (N) = N_Op_Concat then
-         return Do_Op_Concat (N);
+         return Report_Unhandled_Node_Irep (N, "Do_Operator_General",
+                                            "Concat unsupported");
```

Comment:
??? Wasn't the point of this to support `Op_Concat`?

Reply:
Not as a general operator but only as the rhs of array assignment. The extension is relatively simple and will be in the next PR.